python/ciqueue/__init__.py

```diff
@@ -0,0 +1,3 @@
+from ciqueue.static import Static
```

Comment:
I just didn't wan't to expose the internal file structure.

I'd agree for the `import distributed` though, since it has a dependency on `redis`, it's probably better not to import it by default